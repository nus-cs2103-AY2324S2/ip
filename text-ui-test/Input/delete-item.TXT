todo test
deadline asldn /by something
event meeting /from 2pm /to 4pm
list
delete 2
list
bye
