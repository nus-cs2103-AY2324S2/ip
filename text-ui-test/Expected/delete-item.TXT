Hello from
 ______     ______     __  __
/\  __ \   /\  __ \   /\ \/ /
\ \ \/\ \  \ \  __ \  \ \  _-.
 \ \_____\  \ \_\ \_\  \ \_\ \_\
  \/_____/   \/_/\/_/   \/_/\/_/

----------------------------------------------
Welcome! I'm Professor oak
What can I do for you?
----------------------------------------------
Added new Todo: test
----------------------------------------------
----------------------------------------------
Added new Deadline: asldn with Due Date: something
----------------------------------------------
----------------------------------------------
Added new Event: meeting occurring from 2pm to 4pm
----------------------------------------------
----------------------------------------------
1. [T][ ] test
2. [D][ ] asldn (by: something)
3. [E][ ] meeting (from: 2pm to: 4pm)
----------------------------------------------
----------------------------------------------
Are you giving up? Or is this task no longer needed?
Hmmm.. I've deleted Task 1 for you for now.
But, I'll be watching you.
----------------------------------------------
----------------------------------------------
1. [T][ ] test
2. [E][ ] meeting (from: 2pm to: 4pm)
----------------------------------------------
----------------------------------------------
Goodbye! Hope to see you again!
----------------------------------------------
